src/lib/widgets/now-playing/ui/specs/now-playing.spec.ts

```diff
@@ -0,0 +1,9 @@
+export {};
```

Comment:
issue: I think what's also necessary to test here is the presence of the speed toggle and the button to collapse the widget

Reply:
The speed toggle is already included in playback controls, this will be tested in the fourth test. About the collapse button, yes, I will get to it.